db/schema.rb

```diff
@@ -73,6 +73,26 @@
   add_index "news", ["deleted_at"], name: "index_news_on_deleted_at", using: :btree
   add_index "news", ["slug"], name: "index_news_on_slug", unique: true, using: :btree
 
+  create_table "taggings", force: :cascade do |t|
+    t.integer  "tag_id"
+    t.integer  "taggable_id"
+    t.string   "taggable_type"
+    t.integer  "tagger_id"
+    t.string   "tagger_type"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.